src/plugins/dashboard/public/application/dashboard_app_controller.tsx

```diff
@@ -120,6 +126,9 @@ interface UrlParamValues extends Omit<UrlParamsSelectedMap, UrlParams.SHOW_FILTE
   [UrlParams.HIDE_FILTER_BAR]: boolean;
 }
 
+const getSearchSessionIdFromURL = (history: History): string | undefined =>
```

Comment:
Nit: This would probably make more sense inside `src/plugins/dashboard/public/url_generator.ts`. We could add a test for it there too.

Reply:
I am keeping it separate because it depends on client side `history` and  url_generator stuff shouldn't depend on it and should be moved to common eventually.